adafruit_gc_iot_core.py

```diff
@@ -354,7 +354,7 @@ class Cloud_Core:
     """CircuitPython Google Cloud IoT Core module.
 
     :param ESP_SPIcontrol esp: ESP32SPI object.
-    :param dict secrets: Secrets.py file.
+    :param dict secrets: dictionary of settings.
```

Comment:
Should this be renamed to settings?

Reply:
I thought about that, but if people are passing in named args it would break. If you are okay with that change, I'll totally update it.

Reply:
Since `secrets.py` is not mentioned, yes, seems like ok to call it `secrets` still.